include/config/config_debug.h

```diff
@@ -34,4 +37,45 @@
 #define BOOT_CUTSCENE 0x0000
 #define BOOT_PLAYER_NAME 0x15, 0x12, 0x17, 0x14, 0x3E, 0x3E, 0x3E, 0x3E // "LINK"
 
+/**
+ * Disable every debug features
+ */
+// #define DISABLE_DEBUG_FEATURES
+
+// -------------------------------------------
+
+// TODO: find a better way to do that
```

Comment:
About that: initially I put the undefs in the safeguard file, but for the global toggle define I also needed to undef everything, so to avoid duplicates I basically moved the safeguards for everything I added in the config_debug file, also it was easier to enable the "file 1 normal" thing